### ISSUE TITLE
Add "project weight" to export of QF round data button in admin bro - tunable QF matching

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-dapps-v2#3687]
Polygon has given us a list of projects that should get a boost on matching funds via "tunable QF"... you can see this list of names on the 2nd tab [here](https://docs.google.com/spreadsheets/d/1D0_Srinmq9cLDw9zl9GImLIWijyyEeOcFLK9z54t8Qk/edit#gid=0)

I'm working with Algene to check to see if we can grab the slugs corresponding to those "names from polygon" so we can use this in our backend.

Anyway, basically if a project is on that list, any eligible donation should count as 2x the size when we are determining the final matching.

It's possible we could just do this in a spreadsheet, but it's not really easy work.

It would be good to make this something we could perhaps configure in the backend... input a list of projects as mark them as like "add tunable QF"... and then we input the factor by which donations should be multiplied... and then when we fetch the matching funds date (e.g. through the button we added in admin bro in Turkey), that 2x factor on eligible donations is considered.

@CarlosQ96 @mohammadranjbarz @aminlatifi @GriffGreen @jainkrati - it would be good to discuss this and see how much work would be required... and if it makes more sense to use a spreadsheet or to do it in the backend.

**Comment by jainkrati** [comment]
@aminlatifi would you be able to take a look

**Comment by laurenluz** [comment]
one other option we could do is us what Ygg built for the TEC to do the calculations... but I'd also need a dev to help me.

Here is the readme that explains how to add a round: https://github.com/CommonsBuild/tqf#adding-a-new-round

**Comment by aminlatifi** [comment]
@laurenluz I studied the code that has been implemented, and realized the code snippet that calculates the estimated matching is hard-coded in our SQL queries! Implementing this would need days of development and testing.
On the other hand, based on the requirement, I think applying the requested modification to the Polygon QF Round result would be straightforward.
What we need to do is to extract the matching result from admin bro as we used to, and in the spread-sheet do a little calculation.

To understand how the requested change affects the result, consider the QF formula which calculates the weight of a project in non-modified form:
```math
\mathbf{F}_p = \left( \sum_{i=1}^{n} \sqrt{c_{ip}} \right)^2
```
So, for boosted projects we would have
```math
\mathbf{F}_p = \left( \sum_{i=1}^{n} \sqrt{2 \times c_{ip}} \right)^2= \left( \sum_{i=1}^{n} \sqrt{2} \times \sqrt{ c_{ip}} \right)^2 = \left(\sqrt{2} \times \sum_{i=1}^{n} \sqrt{ c_{ip}}\right)^2 = 2 \times \left( \sum_{i=1}^{n} \sqrt{ c_{ip}}\right)^2 
```
Therefore, the requested change will finally end up doubling the project weight in the QF program. I would say, let's extract QF round matching result as always to a spread-sheet. Then, double the values of boosted projects and finally normalize all projects matching to confine the sum of values to the QF program budget. It's achievable in one or two hours of work, while applying your requested changes would need days or weeks.

**Comment by GriffGreen** [comment]
Looks good to me, I think we end up with a spreadsheet anyway, so we just need that "project weight" value in the spreadsheet and we can play with it.

**Comment by laurenluz** [comment]
@mohammadranjbarz & @CarlosQ96 made a button in admin bro "get qf round data" that exports a bunch of values to a spreadsheet... [this one](https://docs.google.com/spreadsheets/d/1Wk8_bG75uQGZnVywos2Ii2t-4QosHoC0pRkMlZ85Ga8/edit#gid=1770697429) - @aminlatifi I just made you an editor so you can see the sheet.

Can one of you help to add the "project weight" as something we export then? in this spreadsheet?

**Comment by aminlatifi** [comment]
@laurenluz For sure, I can do that. There is one point, can a boosted project matched value go upper than limit? if yes the modification would be quite easy... otherwise I need a little time to do the math.

**Comment by laurenluz** [comment]
yes! as we discussed, the boosted projects still should *not* be able to go over the matching cap

**Comment by laurenluz** [comment]
@aminlatifi the list of projects that should get the 2x boost are on the 1st tab of this spreadsheet called "Final jokerace project list"

**Comment by jainkrati** [comment]
if more than 1 hour then pls assign to carlos and share context.

**Comment by mohammadranjbarz** [comment]
@jainkrati @laurenluz  Should we need to implemenet something or just playing with spreadsheets?
@CarlosQ96 Can you take this?

**Comment by CarlosQ96** [comment]
ok ill take it and add the weight to the export.

**Comment by laurenluz** [comment]
yay! thanks @CarlosQ96 !!

**Comment by jainkrati** [comment]
@maryjaf @laurenluz could you please test this.

**Comment by maryjaf** [comment]
This field has been added to the spreadsheet
<img width="1364" alt="image" src="https://github.com/Giveth/giveth-dapps-v2/assets/111529185/8cfd7b59-b2ca-46ad-b6cb-08f6722a70cd">
And for QF polygon round @laurenluz  could approve  that the matching funds was calculated correctly for those  projects

**Comment by maryjaf** [comment]
could you please tell me how value of this field  be set @CarlosQ96 
<img width="1164" alt="image" src="https://github.com/Giveth/giveth-dapps-v2/assets/111529185/d6196ffb-4ef2-467a-a33e-184a74439e1c">

**Comment by jainkrati** [comment]
@CarlosQ96 pls unblock @maryjaf

**Comment by CarlosQ96** [comment]
> could you please tell me how value of this field be set @CarlosQ96 <img alt="image" width="1164" src="https://private-user-images.githubusercontent.com/111529185/305740869-d6196ffb-4ef2-467a-a33e-184a74439e1c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg2MDk0MjUsIm5iZiI6MTcwODYwOTEyNSwicGF0aCI6Ii8xMTE1MjkxODUvMzA1NzQwODY5LWQ2MTk2ZmZiLTRlZjItNDY3YS1hMzNlLTE4NGE3NDQzOWUxYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyMlQxMzM4NDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NzdkNGI0YTA4NjVjNjg4NTAwZjg3NDE0ZDI2YWVkZTk1YjhkZWFhMDI3OGNjOGIxNjlmNWQ2ZWEyZGI4YzhkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7jwVb5YaoTzXzI98Gd0E15JqebW54LTVdq5VXYu9ZS0">

This is part of the QF round formula @maryjaf, its the square root sum squared of the project donations

**Comment by jainkrati** [comment]
@maryjaf to test and close

**Comment by maryjaf** [comment]
> And for QF polygon round @laurenluz could approve that the matching funds was calculated correctly for those projects

Kindly reminder @laurenluz

**Comment by laurenluz** [comment]
ok I tested it on the giving season round data. seems fine.
